Dispatchfile

```diff
@@ -15,15 +15,25 @@ dindTask("dispatch-integration-test",
                  image = "mesosphere/dispatch-dind:v0.5.2",
                  workingDir="/workspace/src-git",
                  args=["git", "fetch", "origin", "master"]),
+
              v1.Container(
                  name = "dispatch-integration-test",
-                 image = "mesosphere/kubeaddons-ci:dispatch",
+                 image = "mesosphere/kubeaddons-ci:dispatch@sha256:80a6f3c8775133b4ad2fab3d44ab4fbb4b541dbc03fcdb296b13f67611427d8c",
                  command = ["make","test"],
                  workingDir = "/workspace/src-git",
                  env = [k8s.corev1.EnvVar(name = "DISPATCH_CI", value = "true"),
+		        k8s.corev1.EnvVar(name = "AWS_REGION", value= "us-west-2"),
```

Comment:
Should we do the same as https://github.com/mesosphere/konvoy/pull/1450 and choose the region randomly to reduce the dependency on a single region in case there's another resource restriction?

Reply:
In the above PR, the picking of a random region is done in the `go` code. `initAWSCISettings() *AWSCISettings `

Here - the region is picked similar to TeamCity, which will run on a) agent region or b) use `AWS_REGION`.  If have to do any round-robin here, we might have to come up with something creative.

--- 

At the moment, I think, using `AWS_REGION` seems okay to me. However, I could think of how an abstraction layer can be added and we can include it for this or other test suites.